electrode-ota-server-dao-mariadb/tsconfig.json

```diff
@@ -37,8 +37,8 @@
     // "paths": {},                           /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
     // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
     // "typeRoots": [],                       /* List of folders to include type definitions from. */
-    "types": ["@types/node", "@types/mocha", "@types/sinon"],                           /* Type declaration files to be included in compilation. */
-    // "allowSyntheticDefaultImports": true,  /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */
+    "types": ["@types/node", "@types/mocha", "@types/semver", "@types/sinon"],                           /* Type declaration files to be included in compilation. */
+    "allowSyntheticDefaultImports": true,  /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */
```

Comment:
why do u need this on?

Reply:
`@types/semver` doesn't have default export, hence we need `allowSyntheticDefaultImports` to be `true` to avoid compilation errors.

Reply:
`import * as React from 'react' ` or  `import { Component } from 'react'`

Reply:
updated the import